$NetBSD$

Don't allow source and license files installing.
Also change destination for headers of i2pd library

--- build/CMakeLists.txt	2018-01-30 16:14:29.000000000 +0000
+++ build/CMakeLists.txt
@@ -466,10 +466,10 @@ if (WITH_BINARY)
   endif ()
 endif ()
 
-install(FILES ../LICENSE
-  DESTINATION .
-  COMPONENT Runtime
-  )
+#install(FILES ../LICENSE
+#  DESTINATION .
+#  COMPONENT Runtime
+#  )
 # Take a copy on Appveyor
 install(FILES "C:/projects/openssl-$ENV{OPENSSL}/LICENSE"
   DESTINATION .
@@ -478,8 +478,8 @@ install(FILES "C:/projects/openssl-$ENV{
   OPTIONAL                      # for local builds only!
   )
 
-file(GLOB_RECURSE I2PD_SOURCES "../libi2pd/*.cpp" "../libi2pd_client/*.cpp" "../daemon/*.cpp" "../build" "../Win32" "../Makefile*")
-install(FILES ${I2PD_SOURCES} DESTINATION src/ COMPONENT Source)
+#file(GLOB_RECURSE I2PD_SOURCES "../libi2pd/*.cpp" "../libi2pd_client/*.cpp" "../daemon/*.cpp" "../build" "../Win32" "../Makefile*")
+#install(FILES ${I2PD_SOURCES} DESTINATION src/ COMPONENT Source)
 # install(DIRECTORY ../ DESTINATION src/
 #   # OPTIONAL
 #   COMPONENT Source FILES_MATCHING
@@ -488,7 +488,7 @@ install(FILES ${I2PD_SOURCES} DESTINATIO
 #   )
 
 file(GLOB I2PD_HEADERS "../libi2pd/*.h" "../libi2pd_client/*.h" "../daemon/*.h")
-install(FILES ${I2PD_HEADERS} DESTINATION src/ COMPONENT Headers)
+install(FILES ${I2PD_HEADERS} DESTINATION include/libi2pd COMPONENT Headers)
 # install(DIRECTORY ../ DESTINATION src/
 #   # OPTIONAL
 #   COMPONENT Headers FILES_MATCHING
@@ -499,7 +499,7 @@ install(FILES ${I2PD_HEADERS} DESTINATIO
 set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Purple I2P, a C++ I2P daemon")
 set(CPACK_PACKAGE_VENDOR "Purple I2P")
 set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/../README.md")
-set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/../LICENSE")
+#set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/../LICENSE")
 file(READ ../libi2pd/version.h version_h)
 string(REGEX REPLACE ".*I2PD_VERSION_MAJOR ([0-9]+).*" "\\1" CPACK_PACKAGE_VERSION_MAJOR "${version_h}")
 string(REGEX REPLACE ".*I2PD_VERSION_MINOR ([0-9]+).*" "\\1" CPACK_PACKAGE_VERSION_MINOR "${version_h}")
@@ -519,11 +519,11 @@ cpack_add_component(Libraries
   DESCRIPTION "Binary libraries for development"
   INSTALL_TYPES full dev3rd
   )
-cpack_add_component(Source
-  DISPLAY_NAME "Source code"
-  DESCRIPTION "I2pd source code"
-  INSTALL_TYPES full
-  )
+#cpack_add_component(Source
+#  DISPLAY_NAME "Source code"
+#  DESCRIPTION "I2pd source code"
+#  INSTALL_TYPES full
+#  )
 cpack_add_component(Headers
   DISPLAY_NAME "Header files"
   DESCRIPTION "I2pd header files for development"
